.github/workflows/release.yml

```diff
@@ -131,6 +131,8 @@ jobs:
             echo $DIFF_INDEX | grep Cargo.lock
             false
           fi
+      - name: Ensure benchmarking compiles
+        run: cargo build --release --features=runtime-benchmarks
```

Comment:
If we aren't actually running the benchmarks we could `cargo check` to make it faster. No strong opinion on my part.

Reply:
I'm in favor, it is faster than building and we aren't running the benchmarks.

Reply:
It would be nice to know that the benchmarks aren't failing at runtime, which could be accomplished by running them as unit tests (`cargo test --features runtime-benchmarks`) or by running the actual benchmarks with low `--steps` and `--repeat` values.

But I think this is a good compromise (assuming `cargo check` is faster).